DashAI/back/dataloaders/params_schemas/jsondataloader.json

```diff
@@ -1,118 +1,129 @@
+          "type": "list_of_strings",
+          "default": []
         }
+      ]
     },
-    "splits": {
-        "display": "div",
-        "properties": {
-            "train_size": {
-                "oneOf": [
-                    {
-                        "error_msg": "The size of the training set must be between 0 and 1.",
-                        "description": "The training set contains the data to be used for training a model. Must be defined between 0 and 100% of the data.",
-                        "type": "float",
-                        "default": 0.7,
-                        "minimum": 0,
-                        "maximum": 1
-                    }
-                ]
-            },
-            "test_size": {
-                "oneOf": [
-                    {
-                        "error_msg": "The test set size must be between 0 and 1.",
-                        "description": "The test set contains the data that will be used to evaluate a model. Must be defined between 0 and 100% of the data.",
-                        "type": "float",
-                        "default": 0.2,
-                        "minimum": 0,
-                        "maximum": 1
-                    }
-                ]
-            },
-            "val_size": {
-                "oneOf": [
-                    {
-                        "error_msg": "The validation set size must be between 0 and 1.",
-                        "description": "The validation set contains the data to be used to validate a model. Must be defined between 0 and 100% of the data.",
-                        "type": "float",
-                        "default": 0.1,
-                        "minimum": 0,
-                        "maximum": 1
-                    }
-                ]
+    "splits_in_folders": {
+      "oneOf": [
+        {
+          "error_msg": "Need to select 'true' if you have folders that define the splits, otherwise 'false'.",
+          "description": "If your data has folders that define the splits select 'true', otherwise 'false'.",
+          "type": "boolean",
+          "default": false
+        }
+      ]
+    }
+  },
+  "splits": {
+    "display": "div",
+    "properties": {
+      "train_size": {
+        "oneOf": [
+          {
+            "error_msg": "The size of the training set must be between 0 and 1.",
+            "description": "The training set contains the data to be used for training a model. Must be defined between 0 and 100% of the data.",
+            "type": "float",
+            "default": 0.7,
+            "minimum": 0,
+            "maximum": 1
+          }
+        ]
+      },
+      "test_size": {
+        "oneOf": [
+          {
+            "error_msg": "The test set size must be between 0 and 1.",
+            "description": "The test set contains the data that will be used to evaluate a model. Must be defined between 0 and 100% of the data.",
```

Comment:
Same here: 
"Percentage of the data that will be assigned to produce the test dataset (in the range 0 to 1, where 0 is 0% and 1 is 100%)."